site/docs/v1/tech/apis/applications.adoc

```diff
@@ -65,6 +66,72 @@ include::docs/v1/tech/apis/_standard-post-response-codes.adoc[]
 
 include::docs/v1/tech/apis/_application-response-body.adoc[]
 
+== Create a New Application From an Existing Application
+
+This API is used to create an Application from an existing Application.
+
+Specifying an Id on the URI will instruct FusionAuth to use that Id when creating the Application. Otherwise, FusionAuth will generate an Id for the Application.
```

Comment:
```suggestion
Specifying an Id on the URI will instruct FusionAuth to use that Id when creating the Application. Otherwise, FusionAuth will generate an Id for the new Application.
```

Reply:
I think we have "create X" hundreds of places in the docs, I think saying "new" is redundant. It is impossible to create an old application. 😎